src/foam/core/crunch/lite/CapableDAO.js

```diff
@@ -42,6 +43,53 @@ foam.CLASS({
   ],
 
   methods: [
+    {
+      name: 'find_',
+      javaCode: `
+        Capable capable = (Capable) getDelegate().find_(x, id);
+        if ( capable == null ) {
+          return null;
+        }
+        capable = (Capable) ((FObject) capable).fclone();
```

Comment:
cloning could be done in populatePayloads if/when the array is not empty.

Reply:
I did it here to follow the same pattern as select, otherwise there would be two clones for select. I'm okay either way. Is there a way i can check for frozen before setting the payloads?

Reply:
FObject.isFrozen()